### PR TITLE
Use add-on installation as a wrapper around add-ons when installed in…

### PR DESCRIPTION
**Pull request by vkareh**
… a cluster

This change is needed in the SDK to fix build failures in https://gitlab.cee.redhat.com/service/uhc-clusters-service/merge_requests/1428

### CODE REVIEW

File: model/clusters_mgmt/v1/add_on_installation_type.model [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an add-on installation in a cluster.
+class AddOnInstallation {
+	// ID used to identify the cluster that this add-on is attached to.
+	ClusterID String
```
Comment (jhernand) [inline model/clusters_mgmt/v1/add_on_installation_type.model]: This should be a link to the cluster: `link Cluster Cluster`.

File: model/clusters_mgmt/v1/add_on_installation_type.model [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	// Link to add-on attached to this cluster.
+	link Addon []AddOn
```
Comment (jhernand) [inline model/clusters_mgmt/v1/add_on_installation_type.model]: This is a a link to a list of add-ons, but should be a link to a single add-on: `link Addon AddOn`.

File: model/clusters_mgmt/v1/add_on_installation_type.model (hunk 1 above)
Comment (vkareh) [reply model/clusters_mgmt/v1/add_on_installation_type.model]: Makes sense. Done

File: model/clusters_mgmt/v1/add_on_installation_type.model (hunk 2 above)
Comment (vkareh) [reply model/clusters_mgmt/v1/add_on_installation_type.model]: Good catch. Fixed.

**Pull request merged**